src/components/property-detail/hero/RequestInspectionPopupScreen.js

```diff
@@ -0,0 +1,252 @@
+import styled from "styled-components";
+import BrandingBar from "../BrandingBar";
+
+const Container = styled.div`
+  position: fixed;
+  background: #00000098;
+  width: 100%;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  z-index: 10002;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const CloseButton = styled.button`
+  border: none;
+  background-color: transparent;
+  color: white;
+  padding: 6px 0;
+  &:hover {
+    cursor: pointer;
+  };
+`;
+
+const SubContainer = styled.div`
+  position: relative;
+  width: 450px;
+  background-color: white;
+`;
+
+const ContentContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px 40px 15px;
+`;
+
+const Title = styled.h2`
+  font-family: "PangeaRegular";
+  font-size: 16px;
+  color: rgb(51, 63, 72);
+  margin: 0;
+`;
+
+const Address = styled.div`
+  font-family: "PangeaRegular";
+  font-size: 12px;
+  font-weight: 300;
+  color: rgb(97, 97, 97);
+  margin: 8px 0 25px 0;
+`;
+
+const SubTitle = styled.h3`
+  font-family: "PangeaRegular";
+  font-size: 13px;
+  font-weight: 300;
+  color: rgb(51, 63, 72);
+  width: 100%;
+  margin: 0;
+
+`;
+
+const RemarksTitle = styled.div`
+  font-family: "PangeaRegular";
+  font-size: 11px;
+  font-weight: 300;
+  color: rgb(97, 97, 97);
+`;
+
+const DayAndTimeContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin-top: 8px;
+`;
+
+const Date = styled.input`
+  width: 38%;
+  border-radius: 4px;
+  border-color: rgb(195, 200, 206);
+  border-style: solid;
+  border-width: 2px;
+  padding: 13px 16px;
+  font-size: 13px;
+  font-weight: 300;
+  color: rgb(155, 155, 155);
+  letter-spacing: 0.6px;
+  text-align: left;
+`;
+
+const Time = styled.input`
+  width: 38%;
+  border-radius: 4px;
+  border-color: rgb(195, 200, 206);
+  border-style: solid;
+  border-width: 2px;
+  padding: 13px 16px;
+  font-size: 13px;
+  font-weight: 300;
+  color: rgb(155, 155, 155);
+  letter-spacing: 0.6px;
+  text-align: left;
+`;
+
+const CheckboxContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 17px;
+`;
+
+const Checkbox = styled.input`
+  margin-left: 0;
+`;
+
+const CheckboxContent = styled.div`
+  margin-left: 8px;
+`;
+
+const Divider = styled.div`
+  margin: 0 auto;
+  width: 370px;
+  height: 1px;
+  background-color: #D2D5DA;
+  margin: 20px 0;
+`;
+
+const PersonalDetailsInputContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+`;
+
+const PersonalDetailsInput = styled.input`
+  width: 38%;
+  border-radius: 4px;
+  border-color: rgb(195, 200, 206);
+  border-style: solid;
+  border-width: 1px;
+  padding: 13px 16px;
+  font-size: 13px;
+  font-weight: 300;
+  color: rgb(155, 155, 155);
```

Comment:
change the color to `color: #333f48;`
and add the below code to change the placeholder text color
```
  &::placeholder {
    color: #333f4866;
  }
```